### PR TITLE
	diagnose PR

### PR DESCRIPTION
**Pull request by jholbrook64**
commit to make PR to send to Sam to diagnose what I did wrong

### CODE REVIEW

File: _targets.R [hunk 1]
```diff
@@ -18,6 +18,11 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   # TODO: PULL SITE DATA HERE
+  tar_target(wi_data, get_site_data(oldest_active_sites, states[1], parameter[1])),
```
Comment (limnoliver) [inline _targets.R:22]: Now that you've added `get_site_data`, you need to tell `targets` where to find this function, so you'll have to add:

`source("1_fetch/src/get_site_data.r)` to the `_targets.R` file.

**Review by limnoliver — CHANGES_REQUESTED** [review]

**Review by jholbrook64 — COMMENTED** [review]
@limnoliver, here is the corrected script

File: 1_fetch/src/find_oldest_sites.R [hunk 2]
```diff
@@ -22,5 +22,6 @@ find_oldest_site <- function(state, parameter) {
   return(best_site)
 }
 find_oldest_sites <- function(states, parameter) {
+  browser()
```
Comment (limnoliver) [inline 1_fetch/src/find_oldest_sites.R]: Can take this out now? Though it is handy having targets plow through this if you don't specify the callr function.

**Review by limnoliver — APPROVED** [review]
Maybe just take the `browser()` out now, and you're good to go.

**Pull request merged**